manim/mobject/text/code_mobject.py

```diff
@@ -183,6 +191,10 @@ def __init__(
         self.background_stroke_width = background_stroke_width
         self.tab_width = tab_width
         self.line_spacing = line_spacing
+        if warn_missing_font:
+            fonts_list = manimpango.list_fonts()
+            if font not in fonts_list:
+                logger.warning(f"Font {font} not in {fonts_list}.")
```

Comment:
Doesn't `Code` at some point create a `Paragraph` that holds the code? Wouldn't the warning then not already be issued by `Paragraph`?

Reply:
I did not add a check for Paragraph because I couldn't figure out how Paragraph actually *uses* the font. It's not a named parameter in the constructor and I didn't see any obvious uses in the class methods. So it felt more intuitive for me to fail earlier in the Code constructor.

Reply:
Paragraph creates a `Text` with the passed arguments, so actually, if there is a warning for `Text`, there should already be one for `Code` too. See here: https://github.com/ManimCommunity/manim/blob/4b576e0f1de2eb4ad5130c16c323adcd9728e7fe/manim/mobject/text/text_mobject.py#L160